src/stock-choose-date-modal/messages_en.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please remove the messages which are not custom ones, I'm pretty sure we don't need all of them for system to work properly

Reply:
Have you reviewed this comment?